### DISCUSSION TITLE
Práctica de CSS ! :)

### DISCUSSION THREAD

**Discussion by DiegoAlvarezGn** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#393] [Show and tell]
Hola, he visto que algunos comparten sus trabajos y pues es la primera vez que lo hago,  también para poder ver sus opiniones o críticas constructivas, aquí está el link de la página (no es responsive aún) : https://diegoalvarezgn.github.io/CSS/
Y esta es una ss que tomé:
![image](https://user-images.githubusercontent.com/99069544/158092669-47404b28-a3da-46b9-be31-066166ea5d63.png)
Si llegaron hasta aquí, les dejo una hamburguesa 🍔 : )

**Comment by Edgarcastrop** [comment]
Te quedo muy bien. disculpa una duda, como creas el dominio de la pagina web?

**Reply by DiegoAlvarezGn** [reply]
Gracias!
Mira dentro de tu repositorio te vas a Settings > Sección (Code and automation) > Pages 
Ahí ya solo seleccionas la rama donde está tú página, guardas y ya se crea el dominio.

**Comment by MarcosSilvaG** [comment]
quedó muy bien :D. pero yo vengo por la amborguesa
